### ISSUE TITLE
cli: auto-completion shows a system command that does not exist

### ISSUE THREAD

**Issue by aciba90** [canonical/ubuntu-advantage-client#2279]
```bash
lxc launch ubuntu-daily:kinetic dev-k && lxc shell dev-k

$ ua TAB
--debug          --version        attach           collect-logs     detach           enable           help             refresh          status           version          
--help           api              auto-attach      config           disable          fix              list             security-status  system

$ ua system
usage: pro <command> [flags]
argument : invalid choice: 'system' (choose from 'attach', 'api', 'auto-attach', 'collect-logs', 'config', 'detach', 'disable', 'enable', 'fix', 'security-status', 'help', 'refresh', 'status', 'version')

$ ua --version
27.11~22.10.1
```

**Comment by orndorffgrant** [comment]
This is fixed in 27.12 :)

**Comment by renanrodrigo** [comment]
Re-opening, as we realized this is unintended behavior
